release note for dbt-databricks in Cloud (#2476)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/04-Nov-2022/dbt-databricks-unity-catalog-support.md

---
title: "The dbt Cloud + Databricks experience is getting even better"
id: "cloud-databricks-unitycatalog"

> description: "The dedicated dbt-databricks adapter is now on dbt Cloud"

Reviewer: ```suggestion
description: "The dbt-databricks adapter is now available for use with dbt Cloud"
```

> sidebar_label: "Enhancement: Support for Databricks Unity Catalog via dbt-databricks"

Reviewer: ```suggestion
sidebar_label: "Enhancement: Support for Databricks Unity Catalog with dbt-databricks"
```

tags: [Nov-17-2022, v1.1.66.15]
---

> dbt Cloud is the easiest and most reliable way to develop and deploy a dbt project. It helps remove complexity all while delivering more power. We’re thrilled to now offer a simpler Databricks connection experience with support for Databricks’ Unity Catalog and better modeling defaults.

Reviewer: ```suggestion
dbt Cloud is the easiest and most reliable way to develop and deploy a dbt project. It helps remove complexity while also giving you more features and better performance. A simpler Databricks connection experience with support for Databricks’ Unity Catalog and better modeling defaults is now available for your use.
```

> This is great news for the hundreds of Databricks customers already using dbt Cloud with the dbt-spark adapter, as they can now [migrate](https://docs.getdbt.com/guides/migration/tools/migrating-from-spark-to-databricks#migration) their connection to the [dbt-databricks adapter](https://docs.getdbt.com/reference/warehouse-setups/databricks-setup) to unlock the benefits. The [Databricks team](https://www.databricks.com/blog/2022/11/17/introducing-native-high-performance-integration-dbt-cloud.html) is committed to maintaining and improving the adapter over time, so you can be sure the integrated experience will provide the best of dbt and the best of Databricks.

Reviewer: ```suggestion
For all the Databricks customers already using dbt Cloud with the dbt-spark adapter, you can now [migrate](https://docs.getdbt.com/guides/migration/tools/migrating-from-spark-to-databricks#migration) your connection to the [dbt-databricks adapter](https://docs.getdbt.com/reference/warehouse-setups/databricks-setup) to get the benefits. [Databricks](https://www.databricks.com/blog/2022/11/17/introducing-native-high-performance-integration-dbt-cloud.html) is committed to maintaining and improving the adapter, so this integrated experience will continue to provide the best of dbt and Databricks.
```

> Check out our [live blog post](https://www.getdbt.com/blog/dbt-cloud-databricks-experience/) to learn more!

Reviewer: ```suggestion
Check out our [live blog post](https://www.getdbt.com/blog/dbt-cloud-databricks-experience/) to learn more.
```
